component-library/src/app/pages/banner-documentation/banner-documentation.component.html

```diff
@@ -7,8 +7,10 @@
       [config]="headingConfig.interactiveDemoSlugConfig"
     ></app-title-slug-url>
     <p
-      class="demo body3"
-      [innerHTML]="'Banner.InteractiveDemoDesc' | translate"
+      class="body3"
+      [innerHtml]="
+        'General.InteractiveDemoDesc' | translate : { component: 'banner' }
```

Comment:
After discussing with Kris, this approach does not translate the component name during French translations. hence we decided to roll back to individual Translations.